src/components/main/compose/messages/mod.rs

```diff
@@ -241,16 +242,20 @@ pub fn Messages(cx: Scope<Props>) -> Element {
                         conversation_id,
                         message_id,
                     } => {
-                        if current_chat.conversation.id() == conversation_id {
-                            match rg.get_message(conversation_id, message_id).await {
+                        let convo = match state.read().all_chats.get(&conversation_id) {
+                            Some(c) => c.to_owned(),
+                            None => ConversationInfo::default(),
```

Comment:
this seems bad. The MessageSent event ought to only be triggered for an existing conversation. This code assumes it's possible to send a message outside of a conversation and if that happens, creates a default ConversationInfo. I think the previous implementation should be used here.